### PR TITLE
change sphinx doc style

### PR DESCRIPTION
**Pull request by RNAer**
1. improve _argsort()
2.

### CODE REVIEW

File: calour/tests/test_util.py [hunk 1]
```diff
@@ -82,11 +82,6 @@ def test_to_list(self):
         self.assertEqual(util._to_list('test'), ['test'])
         self.assertEqual(util._to_list(range(5)), range(5))
 
-    def test__argsort(self):
```
Comment (amnona) [inline calour/tests/test_util.py:85]: Why delete this test?

**Comment by amnona** [comment]
one small comment :)

**Comment by RNAer** [comment]
doctested. see the function of the code

**Comment by amnona** [comment]
cool. so this is instead of standard unit test... nice :)

**Comment by RNAer** [comment]
yes, and also it demo the usage

**Pull request merged**